src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/resource/ClientCredentialsController.kt

```diff
@@ -11,40 +10,26 @@ import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.stereotype.Controller
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.PutMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.ResponseStatus
 import uk.gov.justice.digital.hmpps.authorizationserver.config.AuthenticationFacade
 import uk.gov.justice.digital.hmpps.authorizationserver.config.trackEvent
-import uk.gov.justice.digital.hmpps.authorizationserver.service.ClientCredentialsService
+import uk.gov.justice.digital.hmpps.authorizationserver.service.SaveClientService
 
 @Controller
 class ClientCredentialsController(
-  private val clientService: ClientCredentialsService,
+  private val saveClientService: SaveClientService,
```

Comment:
Would just call this ClientService? Can re-factor it again at the end possibly if it makes sense